shared/entities/base-data.view.ts

```diff
@@ -0,0 +1,214 @@
+import { ViewColumn, ViewEntity } from "typeorm";
+
+@ViewEntity({
+  name: "base_data_view",
+  expression: `
+WITH country_activity_ecosystem_combinations AS (
+	SELECT 
+	    c.code AS country_code,
+	    a.activity as activity,
+	    e.ecosystem as ecosystem
+	FROM 
+	    (SELECT DISTINCT code FROM countries) c
+	CROSS JOIN 
+        (SELECT 'Restoration'::VARCHAR AS activity UNION ALL 
+         SELECT 'Conservation'::VARCHAR) AS a
+    CROSS JOIN 
+        (SELECT 'Mangrove'::VARCHAR AS ecosystem UNION ALL 
+         SELECT 'Seagrass'::VARCHAR UNION ALL 
+         SELECT 'Salt marsh'::VARCHAR) AS e
+)
+select 
```

Comment:
After going deeper in the calculation analysis, I think, as mentioned in this PR, we are missing the Implementation Labor tab/entity from which we need to retrieve a value based on the actity type and/or activity subtype